### DISCUSSION TITLE
What happened to @now/rust?

### DISCUSSION THREAD

**Discussion by ibraheemdev** [vercel/vercel#5295] [Help] [answered]
Around a year ago, an article came out titled "Introducting Now Rust". The article [can still be found on the wayback machine](https://web.archive.org/web/20190312015430/https://zeit.co/blog/introducing-now-rust). However, the [runtimes doc](https://vercel.com/docs/runtimes) now has no mention of Rust, and links to [mike-engel/now-rust](https://github.com/mike-engel/now-rust), a community runtime. Was `@now/rust` removed from vercel? Why?

**Comment by mcsdevv** [comment, marked as answer]
Hello @ibraheemdev, you can find all available Community Runtimes here: https://vercel.com/docs/runtimes#advanced-usage/community-runtimes

**Comment by ibraheemdev** [comment]
So it is a community runtime now? The blog post seemed to say it was officially supported...

**Reply by mcsdevv** [reply]
It is indeed, it was previously going to be supported by Vercel but is now in the hands of the community instead.

**Reply by ibraheemdev** [reply]
Ah, okay, that makes sense.
